.travis.yml

```diff
@@ -1,9 +1,8 @@
 language: node_js
 node_js:
-  - 0.6
-  - 0.8
```

Comment:
Before switching to `-p`, things weren't that much better:

~~~
$ ls -l md5*
-rw-r--r-- 1 Paul 197121 5996 Aug 31  2016 md5.js
-rw-r--r-- 1 Paul 197121 1837 Aug 31  2016 md5.js.gz

$ ls -l dist/md5*
-rw-r--r-- 1 Paul 197121 5878 Apr 25 00:36 dist/md5.min.js
-rw-r--r-- 1 Paul 197121 2378 Apr 25 00:36 dist/md5.min.js.gz
~~~